Documentation/Contributing.md

```diff
@@ -51,9 +51,9 @@ Root
 ...       
 ```
 The template galleries of the Workbooks tools are organized into categories, like Business Hypotheses, Performance, and Usage. Each category can contain many templates.
-![Image of category view](https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/_assets/CategoryView.png)
+![Image of category view](..//_assets/CategoryView.png)
```

Comment:
Since you're updating this document, new screenshots would be nice

Reply:
done